sonar-project.properties

```diff
@@ -7,4 +7,6 @@ sonar.projectKey=alliander-opensource_short-term-forecasting
 
 # relative paths to source directories. More details and properties are described
 # in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
-sonar.sources=.
\ No newline at end of file
+sonar.sources=openstf
+sonar.python.coverage.reportPath=/github/workspace/coverage.xml
+sonar.tests=test
```

Comment:
Misschien hier ook de unit test folder selecteren: `test/unit`

Reply:
Ok, I added it!